src/builder/pod/volume.rs

```diff
@@ -335,7 +335,7 @@ impl SecretOperatorVolumeSourceBuilder {
             if Some(SecretFormat::TlsPkcs12) != self.format {
                 warn!(format.actual = ?self.format, format.expected = ?Some(SecretFormat::TlsPkcs12), "A TLS PKCS12 password was set but ignored because another format was requested")
             } else {
-                annotations.insert(Annotation::tls_pkcs12_password(&password).unwrap());
+                annotations.insert(Annotation::tls_pkcs12_password(password).unwrap());
```

Comment:
Just trying to understand this. We're putting a passphrase into an annotation?
Maybe it's fine, but this can be visible in various K8s UIs, control-plane logs (maybe).

Reply:
This is just a workaround for applications that only support opening encrypted PKCS#12 stores.

Reply:
@Techassi good point, it is not introduced in this PR.

@nightkr, while we're here, is it a reference to where to find the PKCS12 passphrase? Or the actual secret? 
I can imagine some organisations having a problem with this (unless they have alternatives when using PKCS12 - eg: mounting as a Volume or env var from a Secret).

Reply:
It is the actual password. The password is _only_ used to encrypt the PKCS#12 store between the secret-operator and the application. The password is not used to encrypt any secrets at rest (certificates are stored as unencrypted PEMs in `Secret`s).